src/main/java/com/summitsync/api/location/LocationController.java

```diff
@@ -0,0 +1,58 @@
+package com.summitsync.api.location;
+
+import com.summitsync.api.location.dto.GetLocationDto;
+import com.summitsync.api.location.dto.PostLocationDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/v1/location")
+public class LocationController {
+
+    private final LocationMapper mapper;
+    private final LocationService service;
+
+    @PostMapping()
+    public ResponseEntity<GetLocationDto>createLocation(@RequestBody PostLocationDto dto){
+        Location location=mapper.mapPostLocationDtoToLocation(dto);
```

Comment:
Leerzeichen vor und nach dem = würde es etwas übersichtlicher machen

Reply:
Ich werde irgendwann mal einen formatter in eine CI pipeline einbauen. Dann sollte das automatisch gefixed werden. Aktuell gibt es einige inconsistencies im Formatting